source/onboard/migrating-to-mattermost.rst

```diff
@@ -219,35 +219,6 @@ Debugging imports
 
 The ``mmctl import job show`` shows a detailed error message. If you run into problems which the error message does not help to resolve, your best bet is to use the ``mattermost bulk import`` command. The ``mmctl`` import process does not give you any additional debugging information, even in the Mattermost server logs.
 
-Use the Mattermost web app
```

Comment:
In this section, we need to keep only some of the information under a subheading of **Migrate from Slack**:
- migrating from Slack requires the `mmetl tool and bulk loading tool </onboard/migrating-to-mattermost.html#migrating-from-slack-using-the-mattermost-mmetl-tool-and-bulk-import>`__.
- the 3 steps: generating a Slack export file, importing from Slack, and selecting the Slack export file

All other details, including when things were deprecated and using the CLI to import can safely be removed.

Reply:
I think the latest commit addresses this by reverting the deletion of the web app section. Is that alright?

Reply:
Yes. You're right -- your latest commit removes the depreciated CLI content.